website/docs/aggregation operations/aggregation-stages.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 2
+slug: /aggregation-stages/
+---
+
+# Aggregation stages
+
+Aggregation stages are a series of one or more processes in a pipeline that acts upon the returned result of the previous stage, starting with the input documents.
+
+| Supported aggregation stages | Description                                                                                           |
+| ---------------------------- | ----------------------------------------------------------------------------------------------------- |
+| `$count`                     | Returns the count of all matched documents in a specified query                                       |
+| `$group`                     | Groups documents based on specific value or expression and returns a single document for each group   |
+| `$limit`                     | Passes a limit on the documents passed to the next stage                                              |
```

Comment:
Let's phrase it similar to `skip`? Something like "Limits a specified number of documents and passes the rest to the next stage"? Basically, skip and limit are very similar, but skip cuts off the document from the beginning and limit - from the end